great_expectations/datasource/new_datasource.py

```diff
@@ -483,3 +491,24 @@ def _init_data_connectors(
                 name=name,
                 config=config,
             )
+
+    def __getattr__(self, attr: str):
```

Comment:
How about a couple tests: 1. That tries to access an existing attr with no error message 2. That tries to access a fluent attr on a non fluent datasource and we get the correct error message 3. That tries to access an attr that doesn't exist and gets the appropriate error message?

Reply:
Will do.

Reply:
Done